data/countries/JP.yaml

```diff
@@ -25,90 +30,283 @@ holidays:
           en: January 3rd
           jp: 銀行休業日
         type: bank
+      01-15:
```

Comment:
You should be able to acheive the same effect more succinctly with the substitute attribute set to true:

```yaml
      01-15 and if sunday then next monday:
        name:
          en: Coming of Age Day
          jp: 成人の日
        substitute: true
        active:
          - from: 1948-07-20
            to: 1999-12-31
```

Reply:
We cannot do so because substitute holiday in Japan is firstly enforced at 1973-04-12.

With your definition, 1950-01-16 becomes a public holiday as the substitute holiday. However, is not a public holiday.

Reply:
Ah, ok. Thanks for the clarification.